src/popup/style.css

```diff
@@ -30,6 +30,12 @@ body {
   }
 }
 
+.page-popup {
+  /* hardcoded popup height in Chrome */
+  max-height: 600px;
```

Comment:
This is a kludge to hide the overflow when `.submenu` is tall. I couldn't find a proper CSS solution to limit `.submenu` height. And I didn't want to write JS-based height calc.